blackbox/docs/general/occ.rst

```diff
@@ -34,62 +33,75 @@ on every update.
     cr> refresh table locations;
     REFRESH OK, 1 row affected (... sec)
 
-It's possible to fetch the ``_version`` by selecting it::
+It's possible to fetch the ``_seq_no`` and ``_primary_term`` by selecting
+them::
 
-    cr> select name, id, "_version" from locations
+    cr> select name, id, _seq_no, _primary_term from locations
     ... where kind = 'Star System' order by name asc;
-    +----------------+----+----------+
-    | name           | id | _version |
-    +----------------+----+----------+
-    | Aldebaran      | 4  | 3        |
-    | Algol          | 5  | 3        |
-    | Alpha Centauri | 6  | 3        |
-    | Altair         | 7  | 1        |
-    +----------------+----+----------+
+    +----------------+----+---------+---------------+
+    | name           | id | _seq_no | _primary_term |
+    +----------------+----+---------+---------------+
+    | Aldebaran      | 4  |      11 |             1 |
+    | Algol          | 5  |      12 |             1 |
+    | Alpha Centauri | 6  |      13 |             1 |
+    | Altair         | 7  |       1 |             1 |
+    +----------------+----+---------+---------------+
     SELECT 4 rows in set (... sec)
 
-These ``_version`` values can now be used on updates and deletes.
+These ``_seq_no`` and ``_primary_term`` values can now be used on updates
+and deletes.
 
 .. NOTE::
 
     Optimistic concurrency control only works using the ``=`` operator,
-    checking for the exact ``_version`` your update/delete is based on.
+    checking for the exact ``_seq_no`` and ``_primary_term`` your update/delete
+    is based on.
 
 Optimistic Update
 =================
 
-Querying for the correct ``_version`` ensures that no concurrent update has
-taken place::
+Querying for the correct ``_seq_no`` and ``_primary_term`` ensures that no
+concurrent update and cluster configuration change has taken place::
 
     cr> update locations set description = 'Updated description'
-    ... where id=5 and "_version" = 3;
+    ... where id=5 and "_seq_no" = 12 and "_primary_term" = 1;
     UPDATE OK, 1 row affected (... sec)
 
-Updating a row with a wrong or outdated version number will not execute the
-update and results in 0 affected rows::
+Updating a row with a wrong or outdated sequence number or primary term will
+not execute the update and results in 0 affected rows::
 
     cr> update locations set description = 'Updated description'
-    ... where id=5 and "_version" = 2;
+    ... where id=5 and "_seq_no" = 222 and "_primary_term" = 1;
     UPDATE OK, 0 rows affected (... sec)
 
 Optimistic Delete
 =================
 
 Of course the same can be done when deleting a row::
```

Comment:
```suggestion
The same can be done when deleting a row::
```

Not related to the PR, but `Of course` implies that something is trivial or obvious, which for someone reading the docs to learn something is almost never the case.